include/efiauthenticated.h

```diff
@@ -2,214 +2,208 @@
 #define SHIM_EFIAUTHENTICATED_H
 
 #include <wincert.h>
-//***********************************************************************
-// Signature Database
-//***********************************************************************
-///
-/// The format of a signature database. 
-///
+
+/***********************************************************************
+ * Signature Database
+ ***********************************************************************/
+/*
+ * The format of a signature database.
+ */
 #pragma pack(1)
 
 typedef struct {
-  ///
-  /// An identifier which identifies the agent which added the signature to the list.
-  ///
-  EFI_GUID          SignatureOwner;
-  ///
-  /// The format of the signature is defined by the SignatureType.
-  ///
-  UINT8             SignatureData[1];
+	/*
+	 * An identifier which identifies the agent which added the signature to
+	 * the list.
+	 */
+	EFI_GUID SignatureOwner;
+	/*
+	 * The format of the signature is defined by the SignatureType.
+	 */
+	UINT8 SignatureData[1];
 } EFI_SIGNATURE_DATA;
 
 typedef struct {
-  ///
-  /// Type of the signature. GUID signature types are defined in below.
-  ///
-  EFI_GUID            SignatureType;
-  ///
-  /// Total size of the signature list, including this header.
-  ///
-  UINT32              SignatureListSize;
-  ///
-  /// Size of the signature header which precedes the array of signatures.
-  ///
-  UINT32              SignatureHeaderSize;
-  ///
-  /// Size of each signature.
-  ///
-  UINT32              SignatureSize; 
-  ///
-  /// Header before the array of signatures. The format of this header is specified 
-  /// by the SignatureType.
-  /// UINT8           SignatureHeader[SignatureHeaderSize];
-  ///
-  /// An array of signatures. Each signature is SignatureSize bytes in length. 
-  /// EFI_SIGNATURE_DATA Signatures[][SignatureSize];
-  ///
+	/*
+	 * Type of the signature. GUID signature types are defined in below.
```

Comment:
Is "defined in below" a correct English expression? I think the correct sentence would be just "defined below".